packages/backend-app-api/src/services/implementations/userInfo/userInfoServiceFactory.ts

```diff
@@ -36,14 +47,39 @@ export class DefaultUserInfoService implements UserInfoService {
     if (!internalCredentials.token) {
       throw new Error('User credentials is unexpectedly missing token');
     }
-    const { sub: userEntityRef, ent: ownershipEntityRefs = [] } = decodeJwt(
+    const { sub: userEntityRef, ent: tokenEnt } = decodeJwt(
       internalCredentials.token,
     );
 
     if (typeof userEntityRef !== 'string') {
       throw new Error('User entity ref must be a string');
     }
-    if (
+
+    let ownershipEntityRefs = tokenEnt;
+
+    if (!ownershipEntityRefs) {
+      const userInfoResp = await fetch(
```

Comment:
This must use node-fetch (use the same version as other plugins). https://github.com/backstage/backstage/blob/master/docs/architecture-decisions/adr013-use-node-fetch.md